apps/wakunode2/config.nim

```diff
@@ -473,3 +508,26 @@ proc readValue*(r: var TomlReader, val: var crypto.PrivateKey)
         except CatchableError as err:
           raise newException(SerializationError, err.msg)
 
+
+## Configuration validation
+
+let DbUrlRegex = re"^[\w\+]+:\/\/[\w\/\\\.\:\@]+$"
```

Comment:
Should we also add sqlite to the regex, since we only support that at the moment?

Reply:
This is to check the format. I don't want to tie this regex with the supported DB implementation. Those checks are performed by the `setupDbConnection()` proc (upcoming PR).

Reply:
Okay!